src/einsteinpy/symbolic/einstein.py

```diff
@@ -1,5 +1,112 @@
-from .tensor import Tensor
+import sympy
+
+from einsteinpy.symbolic.ricci import RicciScalar, RicciTensor
+from einsteinpy.symbolic.tensor import Tensor, _change_config
 
 
 class EinsteinTensor(Tensor):
-    pass
+    """
+    Class for defining Einstein Tensor
+    """
+
+    def __init__(self, arr, syms, config="ll", parent_metric=None):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        arr : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+            Sympy Array or multi-dimensional list containing Sympy Expressions
+        syms : tuple or list
+            Tuple of crucial symbols denoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
+        config : str
+            Configuration of contravariant and covariant indices in tensor. 'u' for upper and 'l' for lower indices. Defaults to 'll'.
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
+            Corresponding Metric for the Einstein Tensor.
+            Defaults to None.
+
+        Raises
+        ------
+        TypeError
+            Raised when arr is not a list or sympy Array
+        TypeError
+            syms is not a list or tuple
+        ValueError
+            config has more or less than 2 indices
+
+        """
+        super(EinsteinTensor, self).__init__(arr, config=config)
+        self._order = 2
+        self._parent_metric = parent_metric
+        if isinstance(syms, (list, tuple)):
+            self.syms = syms
+            self.dims = len(self.syms)
+        else:
+            raise TypeError("syms should be a list or tuple")
+        if not len(config) == self._order:
+            raise ValueError("config should be of length {}".format(self._order))
+
+    @property
+    def parent_metric(self):
+        """
+        Returns the Parent Metric, if available.
+        """
+        return self._parent_metric
+
+    @classmethod
+    def from_metric(cls, metric):
+        t_ricci = RicciTensor.from_metric(metric)
+        r_scalar = RicciScalar.from_riccitensor(t_ricci, t_ricci.parent_metric)
+        einstein_tensor = t_ricci.tensor() - 0.5 * metric.tensor() * r_scalar.tensor()
```

Comment:
don't use `0.5`, use `(1/2)` instead. This is because, calculations are symbolic, so it will be better to have then in fractional form instead of decimal form.
Also, change `r_scalar.tensor()` to `r_scalar.expr`. Remember `RicciScalar` holds a single expression which can be obtained by using the property `expr`. It is not a tensor(It is actually a 0-order tensor, but we'll ignore the fact for now). For this, we would need to solve #288 xD